Navigation-/Profile/LoginViewController.swift

```diff
@@ -8,6 +8,9 @@ import UIKit
 
 class LoginViewController: UIViewController, UITextFieldDelegate {
     
+    let userService = CurrentUserservice()
+    let userServiceTest = TestUserService()
```

Comment:
нет, нет и нет, LoginViewController должен иметь единственный userService.
хранить два этих сервиса не верно. ViewController просто должен пользоваться зависимостью, которую ему передают при инициализации или любым другим способом.
`let userService = CurrentUserservice()` тоже удалить
```suggestion
    let userService: UserService
```